packages/css-library/src/stylesheets/utilities.scss

```diff
@@ -1,3 +1,26 @@
+@use 'uswds-core' with (
+  $theme-namespace: (
+    'grid': (
+      namespace: 'vads-grid-',
+      output: true,
+    ),
+    'utility': (
+      namespace: 'u-',
+      output: false,
+    ),
+  ),
+  $theme-utility-breakpoints: (
+    "card": false,
+    "card-lg": false,
+    "mobile": true,
+    "mobile-lg": false,
+    "tablet": true,
+    "tablet-lg": false,
+    "desktop": false,
+    "desktop-lg": true,
+    "widescreen": false
+  ),
```

Comment:
The breakpoints here are used in utilities that have `responsive: true` in their settings. Please note that these breakpoints settings are being ignored by the `usa-layout-grid` import.

Reply:
Does this mean we are setting up to use two different sets of breakpoints, one for `usa-layout-grid` and one for everything else?

Reply:
This set of breakpoints affects utilities when they are generated like we have for all other utilities except `usa-layout-grid`.

Here's a [margin utility](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/src/utilities/margin.scss#L10) using [responsive settings](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/src/utilities/settings.scss#L82-L89).

Since we're importing `usa-layout-grid` and not generating them, I believe this [import](https://github.com/uswds/uswds/blob/develop/packages/usa-layout-grid/src/styles/_usa-layout-grid.scss#L3) is overriding our custom uswds-core settings.